sys-process/oomd/oomd-0.5.0-r1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="oomd is an userspace Out-Of-Memory (OOM) killer for Linux systems."
+HOMEPAGE="https://github.com/facebookincubator/oomd"
+SRC_URI="https://github.com/facebookincubator/oomd/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="systemd"
+DEPEND="dev-libs/jsoncpp:=
+	systemd? ( sys-apps/systemd )"
+RDEPEND="${DEPEND}"
+
+CONFIG_CHECK="~CONFIG_PSI"
+ERROR_PSI="CONFIG_PSI is required for oomd to function"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_feature man)
+		$(meson_feature systemd libsystemd)
```

Comment:
I think the question is, does this **link** against libsystemd.so, or is it just a conditional to install systemd service files?

Reply:
Right. :)

Reply:
I think it links against it for the systemd [plugins](https://github.com/facebookincubator/oomd/blob/master/src/oomd/plugins/systemd/BaseSystemdPlugin.cpp#L20) that communicate over dbus. The [`meson.build`](https://github.com/facebookincubator/oomd/blob/master/meson.build#L74) is rather simple to follow.

Reply:
From what I understand reading the [wiki](https://wiki.gentoo.org/wiki/Project:Systemd/Ebuild_policy) the ebuild should include the `systemd` use flag. The [unit file](https://github.com/facebookincubator/oomd/blob/master/src/oomd/etc/oomd.service.in) seems to be reasonable as well.